dolphin/tests/unit/api/fakes.py

```diff
@@ -154,3 +154,67 @@ def fake_access_info_get_all(context, marker=None, limit=None, sort_keys=None,
 
 def fake_sync(self, req, id):
     pass
+
+
+def fake_volume_get_all(context, marker=None,
```

Comment:
Why not we write generator function to create these? It may help for scale too, isn't it?

Reply:
if we use genrator function , expected result can't be compared. 
intention is to compare expected output based on this known input.